services/QuillLMS/app/controllers/cms/activities_controller.rb

```diff
@@ -83,6 +85,7 @@ def destroy
     @raw_score_options = RawScore.order_by_name
     @grade_band_hash = {}
     @raw_score_options.each { |rs| @grade_band_hash[rs.name] = rs.readability_grade_level(@activity_classification.id) }
+    @readability_grade_band_to_minimum_grade_level_map = Activity::READABILITY_GRADE_LEVEL_TO_MINIMUM_GRADE_LEVEL_MAP
```

Comment:
I think `_map` suffix at the end of the variable name: `@readability_grade_band_to_minimum_grade_level_map` and the constant: `Activity::READABILITY_GRADE_LEVEL_TO_MINIMUM_GRADE_LEVEL_MAP` is redundant since both have the word "to" in the middle.

Reply:
I realize it's older code, but I think it would be worth renaming `@grade_band_hash` to something consistent with the naming of  `@readability_grade_band_to_minimum_grade_level`.